common/djangoapps/student/views.py

```diff
@@ -1032,8 +1033,11 @@ def notify_enrollment_by_email(course, user, request):
 
         except Exception:
             log.error('unable to send course enrollment verification email to user from "{from_address}"'.format(
-                        from_address=from_address), exc_info = True)
-            return JsonResponse({"is_success": False, "error": _("Could not send enrollment email to the user"),})
+                        from_address=from_address),exc_info=True)
```

Comment:
I don't believe we _need_ them; I think it was just a personal/stylistic choice.

We could go either way; you decide.